deployments/k8s/helm/signalfx-agent/templates/configmap-monitorsd.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if (.Values.monitorsd) }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "configmap-name" . }}-monitorsd
+  namespace: {{ template "signalfx-agent.namespace" . }}
+  labels:
+    app: {{ template "signalfx-agent.name" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+    chart: {{ template "signalfx-agent.chart" . }}
+data:
+{{- if .Values.monitorsd }}
```

Comment:
The whole file is wrapped in `if` for this value so this seems redundant.

Reply:
I removed this redundant condition.